plugin-openshift/che-plugin-openshift-ext-client/src/main/java/org/eclipse/che/ide/ext/openshift/client/importapp/ImportApplicationPresenter.java

```diff
@@ -192,7 +177,20 @@ public void onBuildConfigSelected(BuildConfig buildConfig) {
 
     @Override
     public void onProjectNameChanged(String name) {
-        view.enableImportButton((selectedBuildConfig != null && view.getProjecName() != null && !view.getProjecName().isEmpty()));
+        view.enableImportButton(selectedBuildConfig != null & isCheProjectNameValid(view.getProjecName()));
+    }
+
+    private boolean isCheProjectNameValid(String projectName) {
+        if (cheProjectsList != null && cheProjectsList.contains(projectName)) {
```

Comment:
maybe need to init cheProjectsList to be sure it's not null, cause empty list of projects condition allows us to import project